packages/core/strapi/src/commands/utils/helpers.ts

```diff
@@ -120,6 +120,14 @@ const assertCwdContainsStrapiProject = (name: string) => {
   };
 
   try {
+    /**
+     * This is basically a generator function & therefore you can use `npx` to create
+     * a plugin in a non-Strapi project.
+     */
+    if (name === 'plugin:init') {
```

Comment:
The call to assertCwdContainsStrapiProject in the plugin:init is what should be removed, rather than making this exception in here

Reply:
So you think it's okay not to use `runAction` to call said action?